app/lib/house_events_scraper.rb

```diff
@@ -0,0 +1,95 @@
+# frozen_string_literal: true
+
+# HouseEventsScraper is used to load House hearings from the house's website:
+#   https://docs.house.gov/Committee/Calendar/ByWeek.aspx
+# It currently just grabs this week and next week's hearings. We'll need to make
+# this more sophisticated in the future.
+#
+# Example usage:
+#   HouseEventsScraper.run
+
+class HouseEventsScraper
```

Comment:
Later, we could also namespace these and split out into separate class files (`HouseEvents::Scraper`, `HouseEvents::Parser`, `HouseEvents::Sync`, etc) but we still have some time to think about that